### PR TITLE
CIV-9409 Access permissions added so the fields are accessible in CUI

### PR DESCRIPTION
**Pull request by maciejmodzelewski**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-9409

### Change description ###
Access permissions added so the fields are accessible in CUI

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json [hunk 1]
```diff
@@ -66,7 +66,12 @@
       {
         "UserRoles": [
         "APP-SOL-SPEC-PROFILE",
-        "APP-SOL-UNSPEC-PROFILE"
+        "APP-SOL-UNSPEC-PROFILE",
+        "RES-SOL-ONE-SPEC-PROFILE",
+        "RES-SOL-ONE-UNSPEC-PROFILE",
+        "RES-SOL-TWO-SPEC-PROFILE",
+        "RES-SOL-TWO-UNSPEC-PROFILE",
+        "CITIZEN-DEFENDANT-PROFILE"
```
Comment (m-meulendijks-v1) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:74]: Do we want all of these to be CRU?
I wonder if the solicitor ones should be read-only.

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json [hunk 2]
```diff
@@ -84,8 +89,13 @@
...
       {
         "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
+          "APP-SOL-UNSPEC-PROFILE",
           "RES-SOL-ONE-SPEC-PROFILE",
-          "RES-SOL-ONE-UNSPEC-PROFILE"
+          "RES-SOL-ONE-UNSPEC-PROFILE",
+          "RES-SOL-TWO-SPEC-PROFILE",
+          "RES-SOL-TWO-UNSPEC-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE"
```
Comment (m-meulendijks-v1) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:98]: Do we want all of these to be CRU?
I wonder if the applicant ones should be read-only.

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json (hunk 1 above)
Comment (maciejmodzelewski) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:74]: I don't know, to be honest. This is what @LeonardoPalmeiro advised me to do.

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json (hunk 2 above)
Comment (maciejmodzelewski) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:98]: I don't know, to be honest. This is what @LeonardoPalmeiro advised me to do.

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json (hunk 1 above)
Comment (m-meulendijks-v1) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:74]: I'm just not sure we want a respondent to have create/update access to an applicant field, and vice versa.

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json (hunk 2 above)
Comment (m-meulendijks-v1) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:98]: I'm just not sure we want a respondent to have create/update access to an applicant field, and vice versa.

Comment (maciejmodzelewski) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json:96]: Fixed.

**Review by m-meulendijks-v1 — DISMISSED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request merged**